### PR TITLE
Add missing examples to contrib examples section.

### PR DESCRIPTION
**Pull request by fabianp**
Also add flax as dependency to examples, which should fix the error in

https://optax.readthedocs.io/en/latest/_collections/examples/mlp_mnist.html

### CODE REVIEW

File: examples/contrib/README.md [hunk 1]
```diff
@@ -7,5 +7,5 @@ Examples that make use of the `optax.contrib` module.
 :glob:
 :maxdepth: 1
 
-[!README.md]*
+*
```
Comment (vroulet) [inline examples/contrib/README.md:10]: Remove then the comment above

File: pyproject.toml [hunk 2]
```diff
@@ -38,8 +38,8 @@ dependencies = [
 ]
 
 [project.urls]
-homepage = "https://github.com/deepmind/optax"
-repository = "https://github.com/deepmind/optax"
+homepage = "https://github.com/google-deepmind/optax"
```
Comment (vroulet) [inline pyproject.toml:41]: As we are on it, shouldn't we change the authors as 
authors = [
    {name = "Google DeepMind", email = "optax-dev@google.com"},
]
?

Comment (fabianp) [reply pyproject.toml:41]: indeed, done!

File: examples/contrib/README.md (hunk 1 above)
Comment (fabianp) [reply examples/contrib/README.md:9]: good catch!

**Comment by fabianp** [comment]
done, please take another look :-)

**Review by vroulet — APPROVED** [review]

**Pull request closed without merging**